### PR TITLE
VyOS: OSPFv3 Resource Module

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

##### SUMMARY

Add new resource module vyos_ospfv3.
Targets model - ansible-network/resource_module_models#83
[vyos_ospfv3]

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

vyos_ospfv3
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

File: plugins/modules/vyos_ospfv3.py [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+# Using merged
+#
+# Before state:
+# -------------
+#
+# vyos@vyos# run show  configuration commands | grep ospfv3
+#
+#
+- name: Merge the provided configuration with the exisiting running configuration
+      vyos.vyos.vyos_ospfv3:
```
Comment (NilashishC) [inline plugins/modules/vyos_ospfv3.py:131]: Indentation issue?

**Pull request merged**